report_aeroo_sample/report/parser.py

```diff
@@ -7,8 +7,8 @@
 
 from odoo import api, models
 
-class ReportSampleParser(models.AbstractModel):
+class Parser(models.AbstractModel):
```

Comment:
Because of [this](https://github.com/aeroo/aeroo_reports/blob/11.0/report_aeroo/models/report.py#L253) it must be named "Parser"